etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export NB_PORT=${NB_PORT:-8888}
```

Comment:
NB_PORT is the notebook port for the local notebook, not the EG port.  If all defaults are taken, the gateway URL will be the same as the notebook URL.  Hmm, I wonder if that would just work?  I doubt it, but it would be like a loopback mode thing. :smile:

Reply:
same comment as above, will cleanup this

Reply:
I think the reference here is important, this is instantiating the notebook (JupyterLab/Elyra) and if gateway url is passed via env variable it will connect to it in a separate pod